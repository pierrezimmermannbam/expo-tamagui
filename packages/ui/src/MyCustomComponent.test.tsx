import { render, screen } from '@testing-library/react-native';
import { View } from 'react-native';

test('custom component', () => {
  render(<View />);

  expect(screen).toMatchSnapshot();
});
