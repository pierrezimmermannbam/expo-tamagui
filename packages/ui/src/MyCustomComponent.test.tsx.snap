// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`custom component 1`] = `<View />`;
